### ISSUE TITLE
nfpm: generated rpm's are missing the release tag in goreleaser v0.120.0+

### ISSUE THREAD

**Issue by joemiller** [goreleaser/goreleaser#1213]
**Describe the bug**

The ["release tag"](http://ftp.rpm.org/max-rpm/s1-rpm-inside-tags.html) is missing (empty) in RPM's generated by the nfpm builder starting with v0.120.0.

I believe this is a bug introduced in nfpm v1.0.0-beta3 and goreleaser v0.120.0. The nfpm repo indicated the bug should be opened against goreleaser so we're opening it here.

This supposedly violates the RPM spec, though more importantly for us it prevents uploading RPMs to https://packagecloud.io which rejects these RPMs with the error:

```
    repository: RPM package parsing error. We weren't able to read your RPM package, which is very odd. Email us so we can figure out why: support@packagecloud.io
```

Upon emailing support they informed us the rpm we are sending is missing the Release tag. We can confirm this with `rpm -qpi`:

```console
$ curl -sL https://git.io/goreleaser | VERSION=v0.120.2 bash -s -- --snapshot --rm-dist

$ rpm -qpi dist/*rpm | head -n4

Name        : pauditd
Version     : 1.1.6-next
Release     :
Architecture: x86_64
```

**To Reproduce**

It should be possible to reproduce this bug locally from the https://github.com/pantheon-systems/pauditd project:

```console
$ git clone git@github.com:pantheon-systems/pauditd.git
$ cd pauditd
```
Execute a snapshot build with latest goreleaser and inspect the resulting RPM to confirm the missing/empty "release" tag:

```console
$ curl -sL https://git.io/goreleaser | VERSION=v0.120.2 bash -s -- --snapshot --rm-dist

$ rpm -qpi dist/*rpm | head -n4

Name        : pauditd
Version     : 1.1.6-next
Release     :
Architecture: x86_64
```

Working backwards we find that v0.119.0 is the most recent version which generates an RPM with a valid release tag:

```console
$ curl -sL https://git.io/goreleaser | VERSION=v0.119.0 bash -s -- --snapshot --rm-dist

$ rpm -qpi dist/*rpm | head -n4

Name        : pauditd
Version     : 1.1.6_next
Release     : 1
Architecture: x86_64
```

**Expected behavior**

RPM's should include a valid [release tag](http://ftp.rpm.org/max-rpm/s1-rpm-inside-tags.html)

**Comment by caarlos0** [comment]
yeah, seems like we "lost" this if on the new impl:

```
{{- with .Info.Overridables.RPM.Release }}
Release: {{ . }}
{{- else }}
Release: 1
{{- end }}
```

working on fix.

**Comment by caarlos0** [comment]
fixed on v0.120.3

thanks for reporting this @joemiller ! 🚀
